sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1258,6 +1258,14 @@ object SQLConf {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefault(10L)
 
+  val STREAMING_STOP_TIMEOUT =
```

Comment:
I think that's not the Spark way now - Spark doesn't add the representation of unit if `timeConf` is being used.

I might tend to agree it may be confused to figure out the actual unit without looking at the code, in dev's perspective. (Some configurations have second as unit, while other configurations have millisecond as unit.) For end users' perspective they can always specify the value as string with unit, so it makes sense to get rid of unit representation from configuration key.